src/components/FilterForm/FilterForm.js

```diff
@@ -98,26 +98,26 @@ export default function FilterForm({
           <span>Calories</span>
           <CaloriesContainer>
             <div>
-            <label htmlFor="caloriesRangeFrom">From</label>
-            <input
-            id="caloriesRangeFrom"
-              name="caloriesRangeFrom"
-              maxLength="4"
-              placeholder="100"
-              value={caloriesRangeFrom}
-              onChange={e => setCaloriesRangeFrom(e.target.value)}
-            />
+              <label htmlFor="caloriesRangeFrom">From</label>
+              <input
```

Comment:
You could  wrap the label around the input. That way you don't need to write id's and `htmlFor`